Gemfile.lock

```diff
@@ -204,6 +219,17 @@ GEM
       multi_xml (~> 0.5)
       rack (>= 1.2, < 4)
     oj (3.11.0)
+    onc_certification_g10_test_kit (4.1.0)
```

Comment:
I think you forgot to remove this after testing.

Reply:
Indeed - I ran `bundle` after by accident, thanks. Removing the g10 and rerunning